appstate.go

```diff
@@ -223,6 +223,41 @@ func (cli *Client) dispatchAppState(mutation appstate.Mutation, fullSync bool, e
 			Action:       mutation.Action.GetUserStatusMuteAction(),
 			FromFullSync: fullSync,
 		}
+	case appstate.IndexLabelEdit:
+		act := mutation.Action.GetLabelEditAction()
+		eventToDispatch = &events.LabelEdit{
```

Comment:
Should there be a label `BuildLabelEdit` to match these incoming events?

Reply:
> Should there be a label `BuildLabelEdit` to match these incoming events?

yeah there it is, i'll try